src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+class TodoForm extends React.Component {
+  render() {
+    return (
+      <form onSubmit={this.props.addMethod}>
+        {/* Event - onChange */}
+        <input
+          placeholder="Item"
+          onChange={this.props.handleChanges}
+          value={this.props.stateObject.task}
+          id={Date.now()}
```

Comment:
```suggestion
          id="todo-input"
```
Give this a unique id, we don't need it's id to be a date or anything.